compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
Thanks @moxiegirl , that is a very good point, I think your suggestion is clearer and more accurate.

@villlem are you ok to update the flag and message in line with @moxiegirl 's wording please?